src/Microsoft.AspNetCore.Razor.Evolution/Legacy/TokenizerView.cs

```diff
@@ -34,15 +34,6 @@ public bool Next()
 
         public void PutBack(TSymbol symbol)
         {
-            Debug.Assert(Source.Position == symbol.Start.AbsoluteIndex + symbol.Content.Length);
-            if (Source.Position != symbol.Start.AbsoluteIndex + symbol.Content.Length)
```

Comment:
I can't really verify this anymore easily so I had to remove it. I want to introduce a real put-back implementation.